tests/nested/manual/core20-4k-sector-size/task.yaml

```diff
@@ -50,7 +50,11 @@ execute: |
     remote.exec "test -e /sys/kernel/security/tpm0/binary_bios_measurements"
 
     echo "and secure boot is enabled on the nested vm"
-    remote.exec "xxd /sys/firmware/efi/efivars/SecureBoot-8be4df61-93ca-11d2-aa0d-00e098032b8c" | MATCH "00000000: 0600 0000 01\s+....."
+    if remote.exec "command -v xxd"; then
```

Comment:
Nitpick, perhaps we can standardize on `od` instead? The output is GREP friendly and we'd have fewer conditional pieces.

Reply:
+1 on using od that should be available everywhere

Reply:
Good idea.

Reply:
I have forced push a replacement since this is such a small commit.